internal/app.go

```diff
@@ -174,6 +179,16 @@ OPTIONS:
   {{range $index, $option := .VisibleFlags}}{{if $index}}
   {{end}}{{$option}}{{end}}{{end}}
 `
+	cli.VersionPrinter = func(c *cli.Context) {
+		db.Init(utils.DefaultCacheDir())
+		metadata, _ := db.Config{}.GetMetadata()
+		b, _ := json.Marshal(ttypes.VersionInfo{
+			TrivyVersion:           version,
+			VulnerabilityDBVersion: metadata,
+		})
+		fmt.Println(string(b))
```

Comment:
I feel the default output should be human-readable. When we receive `-o json`, we should display it as a json format.

Reply:
This isn't the case today. The output is the same regardless of the `-f` option specified. 

```
$ ./trivy -v -f=json
trivy version v0.5.2-6-gede778f
$ ./trivy -v        
trivy version v0.5.2-6-gede778f
```
I'm not aware of a way where we can parse the `format` flag at initialization time with this CLI framework. I also think this output is simple enough to be human readable even in JSON format and we could avoid logic complexity.

Thoughts, ideas?